my-sms/spec/models/title_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Title, type: :model do
+  subject { title }
+  let(:title) { build(:title, title_params) }
+  let(:title_params) { {} }
+
+  context 'with valid params' do
+    it { should validate_presence_of(:name) }
+  end
+
+  context 'a title with the same name exists' do
+    let(:title) { build(:title, name: 'honorific') }
+    let(:other_title) { create(:title, name: 'honorific') }
+
+    before { other_title }
```

Comment:
You can the `bang` version of let, `let!` and then you won't need this before clause here

ie.
`let!(:other_title) { ... }

Reply:
This runs the `create(...)` straight away. The normal version of `let` is (as you have discovered) lazily loaded and only run once it's specifically called

Reply:
Fixed in 9c1858a113f78d27f678467ebe5a7b6fb3efd5ed

Reply:
can now remove the `before {...}` line